scene/main/node.cpp

```diff
@@ -1868,18 +1868,31 @@ bool Node::has_persistent_groups() const {
 
 	return false;
 }
-void Node::_print_tree(const Node *p_node) {
+void Node::_print_tree_pretty(const String prefix, const bool last) {
 
-	print_line(String(p_node->get_path_to(this)));
-	for (int i = 0; i < data.children.size(); i++)
-		data.children[i]->_print_tree(p_node);
+	String new_prefix = last ? " '-" : " |-"; // I would use utf-8 box drawing characters....
```

Comment:
Why not use UTF-8 box drawing characters? :)

Reply:
They weren't displaying for me in the Godot window. I'm not sure why.

Reply:
Try `String::utf8(box_drawing_char)`

Reply:
Thanks @Noshyaar, that was what I needed to know.